app/src/main/java/chat/rocket/android/contacts/models/Contact.kt

```diff
@@ -11,6 +12,21 @@ class Contact() : Parcelable {
     private var emailAddress: String? = null
     private var isPhone: Boolean = true
     private var username: String? = null
+    private var type:Int?=0
```

Comment:
"type" is obsolete now, right?  Same with getType() and setType()?  Can you remove please?

Reply:
Done